webpack.config.shared.js

```diff
@@ -25,8 +26,7 @@ module.exports = {
         loaders: ['react-hot', 'babel-loader?' + JSON.stringify(babelQuerySettings)],
       },  // to transform JSX into JS
       {
-        test: /\.(svg|ttf|woff|woff2|eot)$/,
-        exclude: /node_modules/,
```

Comment:
I needed to remove this `exclude` line to get the Slick theme to load properly with webpack. I talked about this with @aliceriot, and we both think that _all_ of these `exclude:` lines are unnecessary. Should we maybe remove them in a future pull request?